proto/cosmos/group/v1/types.proto

```diff
@@ -160,6 +160,12 @@ message GroupMember {
   Member member = 2;
 }
 
+// GroupAccountCredential represents the group policy account credentials
+message GroupAccountCredential {
+  bytes address_bytes  = 1;
+  bytes derivation_key = 2;
```

Comment:
Why do we need `derivation_key` here? I thought it was needed in the GroupPolicyInfo message

Reply:
This complicates the migration too, if we remove that, we can simplify the migration as well.

Reply:
Tbh, I am not sure why we need at all. At first, I thought it was for deriving the address directly from the pubkey. However, I was getting imports cycle, so I didn't do it. We can if we move the GroupTable Prefixes to types instead.

Reply:
I think it's needed for ADR-033.